docs/user/icl/general.rst

```diff
@@ -0,0 +1,12 @@
+.. _icl-general-information:
+
+General Information
```

Comment:
Why is this a subpage instead of directly in the ICL landing page?

Reply:
Mostly because that's how it's done in the exercise docs